libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -11,6 +11,8 @@ const CONFIG = {
   icon: '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 133.5 118.1" title="Adobe, Inc."><defs><style>.cls-1 {fill: #eb1000;}</style></defs><g><g><polygon class="cls-1" points="84.1 0 133.5 0 133.5 118.1 84.1 0"/><polygon class="cls-1" points="49.4 0 0 0 0 118.1 49.4 0"/><polygon class="cls-1" points="66.7 43.5 98.2 118.1 77.6 118.1 68.2 94.4 45.2 94.4 66.7 43.5"/></g></g></svg>',
 };
 
+const capitalize = (str) => str.charAt(0).toUpperCase() + str.slice(1);
+
```

Comment:
If we go with authored product name, this should no longer be required

Reply:
Agreed!

Reply:
I think this would still be required as the metadata value is not case sensitive I noticed. Am I missing something ?

Reply:
In my view, this shouldn't be a deal breaker for analysts. We can ask authors to capitalize the product name, but ultimately, as long as the analytics values are consistent for the same prompt (which they should be, since we're using a fragment-like pattern), analysts would be able to get the necessary data. So TL;DR, I vote for this being removed.

Reply:
Asking the authors to capitalize won't actually help because the values are lowercased in [here](https://github.com/adobecom/milo/blob/project-pep/libs/features/webapp-prompt/webapp-prompt.js#L14). I can leave it lowercase, which will be consistent for the analytics, but won't be according to the excel requirements file, which is capitalized.
What do you think ?

Reply:
Talked with analytics and they are ok with lowercase. So I removed that function 👍